### ISSUE TITLE
Dynamic Analysis results not displayed on 7b012c26bc3d747827cd6584c8fd1839d46265fa

### ISSUE THREAD

**Issue by shamefulCake1** [Kitware/CDash#1903]
# Bug report

### Expected Behavior

When submitting "dynamic analysis" results, I expect to be able to click on the number of discovered errors and see by-file results, like it used to be in the past.

### Actual Behavior

I can click on the number of errors, but the table displayed there is empty.

### CDash Version

7b012c26bc3d747827cd6584c8fd1839d46265fa

### Additional Information

Postgresql, if that matters.
"Production" build with ssl manually disabled.

**Comment by williamjallen** [comment]
Are there any errors in the logs?

**Comment by shamefulCake1** [comment]
No. Well, at least not in the docker logs for cdash and cdash_postgres.

**Comment by shamefulCake1** [comment]
I wouldn't mind debugging this properly, but cdash uses just too many web trickery I am not aware of.

I found that .vue file with the page, `./resources/js/components/ViewDynamicAnalysis.vue`, but I don't seem to find out how that `cdash.defecttypes` is populated. (What is that "cdash" object? A javascript object? A php object? Do they have a 1-to-1 mapping?

**Comment by williamjallen** [comment]
> No. Well, at least not in the docker logs for cdash and cdash_postgres.

The relevant logs are probably in `/cdash/storage/logs` if you're using our default Docker image.

**Comment by shamefulCake1** [comment]
This directory is empty on both cdash, and cdash_worker. Yes, I am using the default docker image, rebuilt.

**Comment by williamjallen** [comment]
I'm having trouble reproducing this issue locally, and it's hard to debug without more information such as error logs.  I'll leave this issue open in case anyone else happens to be experiencing a similar issue and narrow down the issue to a minimal reproducible example.
